tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -268,6 +269,64 @@ Object.Pipeline {
 			}
 		}
 	}
+
+	gain-capture.1 {
+		format		$FORMAT
+		index		16
+		Object.Widget.pipeline.1 {
+			stream_name	`"DMIC Raw 1"
+		}
+		Object.Widget.copier.1 {
+			stream_name	"DMIC Raw 1"
```

Comment:
Can we not think of a better name than "DMIC Raw 1". "DMIC Raw Stream" or something.

Reply:
sure